### ISSUE TITLE
Link to IPython NB broken

### ISSUE THREAD

**Issue by twiecki** [asmeurer/python3-presentation#1]
http://asmeurer.github.io/python3-presentation/slides.html#68 has a link to:
`http://127.0.0.1:8888/notebooks/examples/Interactive%20Widgets/Image%20Processing.ipynb` at the bottom.

**Comment by asmeurer** [comment]
Yeah, read the text. You have to start the notebook server and run it. As far as I know, the widgets don't work without a running server. 

Maybe I can run a version in Wakari, though.

**Comment by micahhausler** [comment]
If you can't access it from your running notebook, you can find it in the ipython repo https://github.com/ipython/ipython/blob/master/examples/Interactive%20Widgets/Image%20Processing.ipynb

**Comment by asmeurer** [comment]
Sure, but you can only use the widgets if you have a running kernel.

**Comment by micahhausler** [comment]
Correct: the link just didn't work for me. I had to download the notebook and open it in a running kernel.

**Comment by asmeurer** [comment]
I guess the text should be clearer. What you need to do is:
- Clone the IPython git repo
- Run `ipython notebook` from the IPython git repo
- Click on that link, which will open that notebook in the kernel you just started (assuming you used the default port).

**Comment by obilodeau** [comment]
ipython changed a lot (jupyter split, etc.) here are updated locations:

https://github.com/ipython/ipywidgets/blob/0dc1676083527a6d14fd95aa8bc09af67cccfbf8/docs/source/examples/Image%20Processing.ipynb
found via: https://github.com/ipython/ipywidgets/blob/master/docs/source/examples/Index.ipynb
